pkg/apis/kubeone/types.go

```diff
@@ -208,6 +209,33 @@ type PodSecurityPolicy struct {
 	Enable bool `json:"enable"`
 }
 
+// StaticAuditLog feature flag
+type StaticAuditLog struct {
+	Enable bool                 `json:"enable"`
+	Config StaticAuditLogConfig `json:"config"`
+}
+
+// StaticAuditLogConfig config
+type StaticAuditLogConfig struct {
+	// PolicyFilePath is a path on local file system to the audit policy manifest
+	// which defines what events should be recorded and what data they should include.
+	// PolicyFilePath is a required field.
+	// More info: https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#audit-policy
+	PolicyFilePath string `json:"policyFilePath"`
```

Comment:
aren't you hardcoded it later?

Reply:
No, I didn't. As the comment says, this is a file on the local file system (on the system where KubeOne is running). It is hardcoded on instances, but it can be anywhere on the local system.

There were two options: to specify the whole file in the manifest (e.g. like we do for `cloud-config`) or to specify just path and read the file on runtime. After discussion with @eqrx, we decided to go with the latter option because the audit policy file can get very long.

Reply:
OOOOH, it's a reference to the *local* file, on machine where kubeone is running?